tools/common_lib/src/gemm.h

```diff
@@ -376,6 +481,63 @@ class Gemm : public DirectMlBaseNode
         create_operator_impl();
     }
 
+    void record_execute(IDMLCommandRecorder* dml_cmd_recorder, ID3D12GraphicsCommandList* cmd_list, 
```

Comment:
just keep single `record_execute` and pass nullptr for resoucres which are not used

Reply:
Have restructured the code and made Quant Gemm to be a separate class.